backend/config/database/pg/pg.go

```diff
@@ -0,0 +1,129 @@
+package pg
+
+import (
+	"fmt"
+
+	"github.com/jinzhu/gorm"
+
+	"github.com/dwarvesf/smithy/backend/domain"
+)
+
+const (
+	dbUsername      = "postgres"
+	dbName          = "smithy"
+	dbSSLModeOption = "disable"
+	dbPassword      = "example"
+	dbHostname      = "localhost"
+	dbPort          = "5433"
+)
+
+// NewPG new a pg connection
+func NewPG() (*gorm.DB, func(), error) {
+	connectionString := fmt.Sprintf("user=%s dbname=%s sslmode=%s password=%s host=%s port=%s",
+		dbUsername,
+		dbName,
+		dbSSLModeOption,
+		dbPassword,
+		dbHostname,
+		dbPort)
+	db, err := gorm.Open("postgres", connectionString)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return db, func() {
+		db.Close()
+	}, nil
+}
+
+// MigrateTables .
+func MigrateTables(db *gorm.DB) error {
+	err := db.AutoMigrate(
+		domain.User{},
+		domain.Group{},
+		domain.Permission{},
+	).Error
+
+	if err != nil {
+		return err
+	}
+
+	if err := db.Model(&domain.Permission{}).AddForeignKey("user_id", "users(id)", "RESTRICT", "RESTRICT").Error; err != nil {
+		return err
+	}
+	if err := db.Model(&domain.Permission{}).AddForeignKey("group_id", "groups(id)", "RESTRICT", "RESTRICT").Error; err != nil {
+		return err
+	}
+
+	if err := db.Model(&domain.UserGroup{}).AddForeignKey("group_id", "groups(id)", "RESTRICT", "RESTRICT").Error; err != nil {
+		return err
+	}
+	if err := db.Model(&domain.UserGroup{}).AddForeignKey("user_id", "users(id)", "RESTRICT", "RESTRICT").Error; err != nil {
+		return err
+	}
+
+	group1ID := domain.MustGetUUIDFromString("18a5cac3-1024-43c8-acbe-22be76006040")
```

Comment:
there ids should be in constants (ex: SeedGroup1ID)

Reply:
Due to the result returned from `MustGetUUIDFromString` function, I can't assign it to `group1ID` by `const`